core/common/src/main/java/alluxio/util/CommonUtils.java

```diff
@@ -852,5 +861,18 @@ public static boolean isAddressReachable(String hostname, int port) {
     return result;
   }
 
+  private static int initMajorVersion() {
+    String version = System.getProperty("java.version");
+    if (version.startsWith("1.")) {
+      version = version.substring(2, 3);
+    } else {
+      int dot = version.indexOf(".");
+      if (dot != -1) {
```

Comment:
Will there always be a dot?

Reply:
yes, the version string looks like `1.8.1` or '11.0.2`
https://www.oracle.com/java/technologies/javase/versioning-naming.html